test/integration/generated/datatables/src/generatedClientContext.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import { ServiceClient } from "@azure/core-client";
+import { GeneratedClientOptionalParams } from "./models";
+
+/** @internal */
+export class GeneratedClientContext extends ServiceClient {
+  url: string;
+  version: string;
+
+  /**
+   * Initializes a new instance of the GeneratedClientContext class.
+   * @param url The URL of the service account or table that is the target of the desired operation.
+   * @param options The parameter options
+   */
+  constructor(url: string, options?: GeneratedClientOptionalParams) {
+    const defaults: GeneratedClientOptionalParams = {
+      requestContentType: "application/json; charset=utf-8"
+    };
+
+    const optionsWithDefaults = {
```

Comment:
you'll want to mix the XML parse/stringify functions into `deserializationOptions`/`serializationOptions`:

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/tables/data-tables/src/TableClient.ts#L162